src/index.js

```diff
@@ -49,7 +49,13 @@ class FtpServer extends EventEmitter {
       this.connections[connection.id] = connection;
 
       socket.on('close', () => this.disconnectClient(connection.id));
-      socket.once('close', () => this.emit('disconnect', {connection, id: connection.id}));
+      socket.once('close', () => {
+        let newConnectionCount = Object.keys(this.connections).length;
+        this.emit('disconnect', {connection, id: connection.id, newConnectionCount: newConnectionCount});
```

Comment:
```suggestion
        this.emit('disconnect', {connection, id: connection.id, newConnectionCount: Object.keys(this.connections).length});
```

Single use variable 🤷🏻